src/utils/purgatory.ts

```diff
@@ -0,0 +1,25 @@
+import { PurgatoryData as PurgatoryDataAsset } from '@oceanprotocol/lib'
+import { fetchData } from '.'
+
+const purgatoryUrl = 'https://market-purgatory.oceanprotocol.com/api/'
+
+export interface PurgatoryDataAccount {
+  address: string
+  reason: string
+}
+
+export default async function getAssetPurgatoryData(
+  did: string
+): Promise<PurgatoryDataAsset> {
+  // const data = await fetchData(`${purgatoryUrl}asset?did=${did}`)
+  // return { did: data[0]?.did, reason: data[0]?.reason }
+  return { did: undefined, reason: undefined }
+}
+
+export async function getAccountPurgatoryData(
+  address: string
+): Promise<PurgatoryDataAccount> {
+  // const data = await fetchData(`${purgatoryUrl}account?address=${address}`)
+  // return { address: data[0]?.address, reason: data[0]?.reason }
+  return { address: undefined, reason: undefined }
+}
```

Comment:
thought(blocking): we should add a `/TODO` comment or similar mentioning why this was commented out

Reply:
Alternatively, I was considering the possibility of adding the purgatory url to the `app.config` file and just adding a url check to the `purgatory.ts` file to evaluate if it should make the api call or not. Do you think it would be better or is it unnecessarly complex?

Reply:
No also thought of this. If it does not take too much time go ahead and add this. Just make sure to comment in the config on how to use the variable (e.g., leave empty to skip purgatory calls or something like that)

Reply:
Should be better now, please have a look.